testing/e2e/suites/osm.go

```diff
@@ -23,14 +42,26 @@ func osmSuite(in infra.Provisioned) []test {
 				withVersions(manifests.AllUsedOperatorVersions...).
 				withZones(manifests.AllDnsZoneCounts, manifests.AllDnsZoneCounts).build(),
 			run: func(ctx context.Context, config *rest.Config, operator manifests.OperatorConfig) error {
-				if err := clientServerTest(ctx, config, operator, osmNs, in,
-					func(ingress *netv1.Ingress, service *corev1.Service, _ zoner) error {
-						annotations := ingress.GetAnnotations()
-						annotations["kubernetes.azure.com/use-osm-mtls"] = "true"
-						ingress.SetAnnotations(annotations)
-
-						return nil
-					}, nil); err != nil {
+				if operator.DisableOsm {
```

Comment:
this isn't needed. it's specified above that osm is required

Reply:
done